core/metadata-3.2.md

```diff
@@ -201,6 +253,8 @@ following labels and formats:
 | 767 | `ERR_KEYINVALID`      | `<Key> :invalid metadata key`            |
 | 768 | `ERR_KEYNOTSET`       | `<Target> <Key> :key not set`            |
 | 769 | `ERR_KEYNOPERMISSION` | `<Target> <Key> :permission denied`      |
+| 770 | `ERR_METADATARATELIMIT` | `<Target> <Key> [<RetryAfter>]`        |
+| 771 | `ERR_METADATASYNCLATER` | `<Target> [<RetryAfter>]`               |
 
```

Comment:
I'd also propose that `ERR_METADATARATELIMIT` should include the value, to make it easier for clients to schedule a retry. Since the value would need to be in the last position, an optional `RetryAfter` would need to be encoded as *.

e.g.

```
# Retry in 5 seconds
:server 775 client_nick target key 5 :value
# No retry value
:server 775 client_nick target key * :value
```